app-emacs/kaolin-themes/kaolin-themes-1.6.9.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit elisp
+
+DESCRIPTION="Set of eye pleasing themes for GNU Emacs"
+HOMEPAGE="https://github.com/ogdenwebb/emacs-kaolin-themes"
+SRC_URI="https://github.com/ogdenwebb/emacs-kaolin-themes/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/emacs-kaolin-themes-${PV}"
```

Comment:
Might be worth defining `MY_PN="emacs-${PN}"` and use the variable for `SRC_URI` and `S`.